app/src/main/res/values/strings.xml

```diff
@@ -9,4 +9,8 @@
 
     <string name="hello_first_fragment">Hello first fragment</string>
     <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
+    <!-- TODO: Remove or change this placeholder text -->
```

Comment:
Are you using `hello_first_fragment `or `hello_second_fragment `right now? If not, remove it.

If you are using it, something is not right with your pr.

Reply:
Removed all references to unused  First and Second Fragments
61ec09ecb3e9ef5f2ff9e2add5e0b7e01deda10e